src/menu/views/file_info.c

```diff
@@ -16,12 +16,17 @@ static const char *music_extensions[] = { "mp3", "wav", "ogg", "wma", "flac", NU
 static const char *controller_pak_extensions[] = { "mpk", "pak", NULL };
 static const char *emulator_extensions[] = { "nes", "smc", "gb", "gbc", "sms", "gg", "chf", NULL };
 static const char *cheat_extensions[] = {"cht", "cheats", "datel", "gameshark", NULL};
-
+static const char *controller_pak_note_extensions[] = { "mpkn", NULL };
```

Comment:
I would like to discuss this new mpkn for notes.... I would like further discussion on whether this the best standard for raw cpaknotes for support with libdragon and ares.